public/js/draw/draw.js

```diff
@@ -389,3 +389,99 @@ function regionClicked(e) {
         prevY = position.y;
     }
 }
+
+
+/**
+ * Convert JSON data into SVG elements then populate the canvas with them.
+ */
+function jsonToSvg(jsonStr) {
+    'use strict';
+
+    var jsonData = JSON.parse(jsonStr);
+
+    // Parse Path data
+    jsonData[2][1].forEach(function(jsonPath) {
+        // Create Paths and Regions
+        var path = document.createElementNS(xmlns, 'path');
+        path.setAttributeNS(null, 'id', 'pathp' + nodeId);
+        path.setAttributeNS(null, 'd', 'M' + jsonPath.points.join(' '));
+        path.setAttributeNS(null, 'data-active', 'inactive');
+        if (jsonPath.isRegion) {
+            path.setAttributeNS(null, 'class', 'region');
+            path.setAttributeNS(null, 'style', 'fill:' + jsonPath.fill);
+        } else {
+            path.setAttributeNS(null, 'stroke', 'black');
+            path.setAttributeNS(null, 'marker-end', 'url(#arrow)');
+            path.setAttributeNS(null, 'class', 'path');
+        }
+
+        svgDoc.appendChild(path);
+        nodeId += 1;
+    });
+
+    // Parse Shape data
+    jsonData[1][1].forEach(function(jsonShape) {
+        // Create Shape and add it to SVG canvas
+        var g = document.createElementNS(xmlns, 'g');
+        var node = document.createElementNS(xmlns, 'rect');
+
+        g.setAttribute('class', 'node');
+        g.setAttribute('id', 'g' + nodeId);
+        g.setAttribute('data-active', 'active');
+        g.setAttribute('fill', jsonShape.fill);
+
+        node.setAttribute('x', jsonShape.pos[0]);
+        node.setAttribute('y', jsonShape.pos[1]);
+        node.setAttribute('rx', 4);
+        node.setAttribute('ry', 4);
+        node.setAttribute('id', 'n' + nodeId);
+        node.setAttribute('width', jsonShape.width);
+        node.setAttribute('height', jsonShape.height);
+        node.setAttribute('class', 'node');
+        node.parents = [];
+        node.kids = [];
```

Comment:
I realize that `kids` is a relic from the creation of the draw tool, but is there a better name for it? Perhaps a name that doesn't necessitate a comment?